taxi/views.py

```diff
@@ -44,6 +65,26 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_form.html"
```

Comment:
no need to override `template name` here. and also fix it in the `manufacter` view's